src/bin/makesrc.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/sh
+# © 2016 and later: Unicode, Inc. and others.
```

Comment:
Also this file, is it a copy of some existing file?

Reply:
it's a copy of one of the sibling scripts in the same directory. Will fix.